src/crud/updateBackTranslationType.ts

```diff
@@ -0,0 +1,68 @@
+import { QueryBuilder, TransformBuilder } from '@orbit/data';
+import MemorySource from '@orbit/memory';
+import { AxiosError } from 'axios';
+import { ArtifactType, MediaFile } from '../model';
+import { UpdateRelatedRecord } from '../model/baseModel';
+import { logError, Severity } from '../utils';
+import { axiosGet } from '../utils/axios';
+
+export const updateBackTranslationType = async (
+  memory: MemorySource,
+  token: string | null,
+  user: string,
+  errorReporter: any
+) => {
+  if (token) {
+    await axiosGet('mediafiles/wbt', undefined, token)
+      // eslint-disable-next-line no-loop-func
+      .then((response) => {
+        var fr = response.data as any;
+        console.log(fr);
```

Comment:
It looks like all we do with the response data is put it on the console log? Do we need a comment or more code here?